Cargo.toml

```diff
@@ -31,12 +32,12 @@ futures = "0.3"
 futures-util = "0.3"
 git-version = "0.3.5"
 hex = "0.4.3"
-iroh-bitswap = { git = "https://github.com/nathanielc/beetle", branch = "remove-p2p" }
-iroh-metrics = { git = "https://github.com/nathanielc/beetle", branch = "remove-p2p" }
-iroh-rpc-client = { git = "https://github.com/nathanielc/beetle", branch = "remove-p2p" }
-iroh-rpc-types = { git = "https://github.com/nathanielc/beetle", branch = "remove-p2p" }
-iroh-store = { git = "https://github.com/nathanielc/beetle", branch = "remove-p2p" }
-iroh-util = { git = "https://github.com/nathanielc/beetle", branch = "remove-p2p" }
+iroh-bitswap = { git = "https://github.com/nathanielc/beetle", branch = "custom-behaviour" }
+iroh-metrics = { git = "https://github.com/nathanielc/beetle", branch = "custom-behaviour" }
+iroh-rpc-client = { git = "https://github.com/nathanielc/beetle", branch = "custom-behaviour" }
+iroh-rpc-types = { git = "https://github.com/nathanielc/beetle", branch = "custom-behaviour" }
+iroh-store = { git = "https://github.com/nathanielc/beetle", branch = "custom-behaviour" }
+iroh-util = { git = "https://github.com/nathanielc/beetle", branch = "custom-behaviour" }
```

Comment:
Why was this changed? The `custom-behaviour` branch is older than the `remove-p2p` branch. In fact we should be using `main`. I'll create a small PR to fix that and then we can rebase this.

Reply:
Feel free to merge once https://github.com/3box/rust-ceramic/pull/65 merges and this is rebased.

Reply:
Will do. `remove-p2p` branch was deleted, so wasn't able to build without switching.